src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValue.ts

```diff
@@ -56,6 +56,11 @@ export class DynamicFacetValue implements IDynamicFacetValue {
     return value.toLowerCase() === this.value.toLowerCase();
   }
 
+  public focus() {
+    const checkbox = this.renderedElement.querySelector<HTMLElement>('.coveo-checkbox-button');
```

Comment:
Should there be optional chaining used here ? Just to prevent the checkbox from being null and throwing.

Reply:
Sure, I'll add it